### DISCUSSION TITLE
出现个问题，在service层定义了一个方法，参数数没问题的情况下却走到了另一个if进去

### DISCUSSION THREAD

**Discussion by mynamexjl** [midwayjs/midway#3942] [Q&A]
### Describe the problem（描述问题）

![ECD0F12C-F823-4932-9F51-18445FCA9660](https://github.com/midwayjs/midway/assets/25628679/cfd26a72-6cef-4c4b-98c5-7c52f4f2b987)
问题描述：线上环境 ： 传递的 openId 与默认 reviewDoc.openId 是不相等但是却走进了相等的if 进去。 预发环境没有这个情况，在线上环境上 添加了 console.log 后数据又正常判断了。 这边出这个问题是因为部署的问题还是什么问题导致的呢。 已经有几次类似这样的情况了，打印了console 后 判断才正常

### Midway Versions（Midway 版本）

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.15.0)
✓ @midwayjs/axios(3.15.1)
✓ @midwayjs/bootstrap(3.15.0)
✖ @midwayjs/bull(current: 3.11.6, allow: ["3.15.0","3.15.1","3.15.2"])
✓ @midwayjs/bull-board(3.15.1)
✓ @midwayjs/cache-manager(not installed)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.15.0)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(3.15.1)
✓ @midwayjs/decorator(3.15.0)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(not installed)
✓ @midwayjs/i18n(3.15.1)
✓ @midwayjs/info(not installed)
✓ @midwayjs/jwt(not installed)
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.15.0)
✓ @midwayjs/mongoose(3.15.1)
✓ @midwayjs/mqtt(not installed)
✓ @midwayjs/oss(3.15.0)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(3.15.1)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(3.15.0)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(3.15.2)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(3.15.1)
✓ @midwayjs/swagger(3.15.1)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/typegoose(3.15.1)
✓ @midwayjs/typeorm(3.15.0)
✓ @midwayjs/upload(3.15.1)
✓ @midwayjs/validate(3.15.1)
✓ @midwayjs/version(3.15.1)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(3.15.0)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.15.2)
✖ @midwayjs/ws(current: 3.16.0, allow: ["3.15.0","3.15.2"])

**Comment by czy88840616** [comment]
无法猜测。

**Comment by waitingsong** [comment]
项目内排查下有没有对 openId 值进行过修改。
或者 reviewDoc 在并发下是不是保持一致性

**Reply by mynamexjl** [reply]
没有 直接就是 控制层后就调用了，没有其他地方有赋值了

**Reply by mynamexjl** [reply]
这边看了日志的请求记录并没有并发的情况
